mitosheet/css/elements/SearchBar.css

```diff
@@ -28,6 +28,19 @@
     justify-content: center;
 }
 
-.mito-close-search-button:hover {
+.mito-search-button:hover {
     background-color: var(--mito-highlight-light);
 }
+
+.mito-grid-cell-search-match {
+    background-color: var(--mito-highlight-light);
+    color: var(--mito-text);
+    font-weight: bold;
+}
```

Comment:
I'm actually not sure we need this. I think just normal section in the sheet is enough to show that you moved to something new - just like in Excel!

In which case, can we remove this class, and remove the logic applying it to cells and headers?